examples/bitcoin_psbt/package.json

```diff
@@ -10,10 +10,10 @@
         "getrawtransaction": ".bitcoin/bin/bitcoin-cli -conf=$(pwd)/.bitcoin.conf getrawtransaction $npm_config_txid",
         "help": ".bitcoin/bin/bitcoin-cli -conf=$(pwd)/.bitcoin.conf help",
         "premanualtest": "ts-node --transpile-only --ignore=false test/pretest.ts",
-        "manualtest": "ts-node --transpile-only --ignore=false test/test.ts",
+        "manualtest": "jest test/manual_test.ts test/manual.test.ts",
```

Comment:
why are command-line arguments being passed into jest here?

Reply:
to limit which tests are run